tasks/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml

```
@@ -286,9 +286,10 @@ spec:
           memory: 256Mi
       image: quay.io/enterprise-contract/ec-cli:snapshot
       onError: continue  # progress even if the step fails so we can see the debug logs
-      command: [cat]
+      command: [sh, -c]
       args:
-        - "$(params.HOMEDIR)/report-json.json"
+        # Format the JSON output to wrap lines at 8000 characters for readability
```

I'm trying to write a better explanation. It's not really readability, since you still can read it.

```suggestion
        # The report can get very large, so add some line breaks rather than
        # render as a single line. This makes it easier to render in the UI,
        # easier to copy/paste, and less likely to cause problems with logging
        # systems or other consumers of the data, (assuming they correctly parse
        # the full output).
```